<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pathPlanner</class>
 <widget class="QWidget" name="pathPlanner">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>287</width>
    <height>58</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>40</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Mission</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="buttonNewMission">
        <property name="toolTip">
         <string>Clear the mission</string>
        </property>
        <property name="text">
         <string>New</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="buttonLoad">
        <property name="toolTip">
         <string>Load a mission</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="../resources.qrc">
          <normaloff>:/:/plugins/PathPlanner_3/icons/load.png</normaloff>:/:/plugins/PathPlanner_3/icons/load.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="buttonSave">
        <property name="toolTip">
         <string>Save the current mission</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="../resources.qrc">
          <normaloff>:/:/plugins/PathPlanner_3/icons/save.png</normaloff>:/:/plugins/PathPlanner_3/icons/save.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
